code/obj/item/generic_box.dm

```diff
@@ -195,6 +195,15 @@
 				attack_hand()
 				attack()
 
+				research //For the research module
+					name = "box shaped artifact form dispensor"
+					desc = "A box full of forms for classifying alien artifacts"
+					icon_state = "item_box"
+					icon_closed = "item_box"
+					icon_open = "item_box-open"
+					set_contained_items()
+						contained_items = concrete_typesof(/obj/item/sticker/postit/artifact_paper)
```

Comment:
```suggestion
						contained_items = list(/obj/item/sticker/postit/artifact_paper)
```
no reason to use `concrete_typesof` here, since you only want one type